packages/utilities/README.md

```diff
@@ -0,0 +1,26 @@
+# @drupal/admin-ui-components
+
+Helper utilities for Admin UI and extension point projects.
+
+## `api`
+
+An `async` helper function for making requests to a Drupal backend.
+
+- `@param {string} REACT_APP_DRUPAL_BASE_URL` The base url of the backend (Drupal)
```

Comment:
Do you mind copying this documentation to the api function itself, so tools could show it?